Snowdog_Components/components/Organisms/order-summary/_order-summary.scss

```diff
@@ -158,4 +158,12 @@
     &__product-price {
         font-size: $order-summary__product-price-font-size;
     }
+
+    &__dropdown-list {
+        background-color: $order-summary__dropdown-list-background-color;
+
+        .dropdown-list__label {
+            background-color: $order-summary__dropdown-list-label-background-color;
+        }
```

Comment:
I think it would be good to use `$order-summary__background` as a value here because we want the background to match the background of order-summary container. I don't think we will ever need to change it to a different color than that container